src/OrchardCore.Modules/OrchardCore.Sitemaps/Views/Admin/Display.cshtml

```diff
@@ -24,25 +24,30 @@
 </div>
 @* do we need antiforgery here ?*@
 <form>
-    @if (Model.Items.Any())
-    {
-        <ul class="list-group">
-            @foreach (var item in Model.Items)
-            {
-                item.SitemapPlanId = Model.Sitemap.SitemapId;
+    <div class="form-group"></div>
```

Comment:
Why opening and closing on the same line?

Reply:
Fixed